corehq/apps/geospatial/templates/geospatial/partials/saved_polygon_filter.html

```diff
@@ -2,17 +2,24 @@
 
 <div class="col-sm-5">
     <div class="row form-horizontal">
-        <label for="saved-polygons" class="control-label col-sm-4">
+        <label for="saved-polygons" class="control-label col-sm-3">
         {% trans "Filter by Saved Area" %}
         </label>
-        <div class="col-sm-6">
+        <div class="col-sm-5">
             <select id="saved-polygons"
                     class="form-control"
                     data-bind="select2: savedPolygons,value: selectedSavedPolygonId,">
             </select>
         </div>
-        <button class="col-sm-2 btn btn-default" data-bind="click: clearSelectedPolygonFilter, visible: selectedSavedPolygonId">
-            {% trans 'Clear' %}
-        </button>
+        <div class="col-sm-1">
+            <button class="btn btn-default" data-bind="click: clearSelectedPolygonFilter, visible: selectedSavedPolygonId">
+                {% trans 'Clear' %}
+            </button>
+        </div>
+        <div class="col-sm-2" style="margin-left:5px">
+            <button class="btn btn-default" data-bind="click: deleteSelectedPolygonFilter, visible: selectedSavedPolygonId">
+                {% trans 'Delete Area' %}
```

Comment:
Might I suggest that we put the bin icon on the button? Or change the colour of the button to red at least, indicating a destructive action?

Reply:
Added in https://github.com/dimagi/commcare-hq/pull/34717/commits/4ac06f93d04538c13b8f0ce97d78e9ba14caa7a6

Reply:
I added the bin icon as well (can see them in data dictionary)